### ISSUE TITLE
Popupbox content showing even if the container is hidden

### ISSUE THREAD

**Issue by nrmohan** [ButchersBoy/MaterialDesignInXamlToolkit#294]
I have a popubox in the mainwindow, when the window is minimized to tray the popup box is showing in the top left of the screen on hover even over another application.

![1](https://cloud.githubusercontent.com/assets/16112725/13803345/df499f5a-eb69-11e5-8c51-5113723fdc14.png)
![2](https://cloud.githubusercontent.com/assets/16112725/13803346/df4a6354-eb69-11e5-8cc7-359fc6a0a034.png)

**Comment by MahmoudSaid356** [comment]
I have the same problem

**Comment by CarHel** [comment]
I tested a bit popupBox-dialoghost that has a popupBox for content. The problem seems fixed, but the problem comes back when there is a delay before the popupbox is shown and in the meantime the main window is not the activated window. I tried to debug it, the problem seems to come from PopupEx. 

`var hwndSource = (PresentationSource.FromVisual(this.Child)) as HwndSource;`
returns null when the popupBox is shown and the mainwindow isn't the activated window.
